x/deployment/keeper/test_common.go

```diff
@@ -27,40 +20,44 @@ func createTestCodec() *codec.Codec {
 	return cdc
 }
 
-// SetupTestInput will setup test inputs and return context and keeper
-func SetupTestInput() (sdk.Context, auth.AccountKeeper, params.Keeper, bank.BaseKeeper, Keeper) {
-	db := dbm.NewMemDB()
+// // SetupTestInput will setup test inputs and return context and keeper
```

Comment:
Note: the requirement to have a codec in these keeper tests is causing an import cycle. Will need to see how other modules in the SDK handle this.

Reply:
we changed all of the keeper tests.  these tests were doing too much setup to just test the keeper.

the import cycle fix is to use the package `keeper_test` instead of `keeper`